# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: 2147c5c81edf
# Full Hash: 2147c5c81edf9cd56c19e7e92dfebfcd77329978
# Author: Ryan VanderMeulen <ryanvm@gmail.com>
# Date: 2023-03-01 17:52 -0500
# Description:
#   Backed out changesets b05343a5b533 and 57b2cda7e0db (bug 1818349) for causing bug 1819723.
# ==============================================================================

diff -r d59b76766f0d -r 2147c5c81edf .cargo/config.in
--- a/.cargo/config.in	Wed Mar 01 05:53:30 2023 +0000
+++ b/.cargo/config.in	Wed Mar 01 17:52:01 2023 -0500
@@ -15,6 +15,11 @@
 rev = "a6514854d4518b02f2805719ff6cd74dae7dfde6"
 replace-with = "vendored-sources"
 
+[source."https://github.com/bendk/application-services"]
+git = "https://github.com/bendk/application-services"
+rev = "ecb35df5fc40357c49922f90e86bf4147fa52953"
+replace-with = "vendored-sources"
+
 [source."https://github.com/chris-zen/coremidi.git"]
 git = "https://github.com/chris-zen/coremidi.git"
 rev = "fc68464b5445caf111e41f643a2e69ccce0b4f83"
@@ -75,11 +80,6 @@
 rev = "688a6574cf830b4a3adaf9c5d41efc1f7dd4017f"
 replace-with = "vendored-sources"
 
-[source."https://github.com/mozilla/application-services"]
-git = "https://github.com/mozilla/application-services"
-rev = "5433b3dc9ac152a0da29ed6642df839d79515a59"
-replace-with = "vendored-sources"
-
 [source."https://github.com/mozilla/audioipc"]
 git = "https://github.com/mozilla/audioipc"
 rev = "fb7a2b12ced3b43e6a268621989c6191d1ed7e39"